app/components/NoticeTypeCheckboxes/NoticeTypeCheckboxes.tsx

```diff
@@ -230,14 +230,12 @@ export function NoticeTypeCheckboxes({
     JsonNoticeTypes.SVOM = ['gcn.notices.svom']
     JsonNoticeTypeLinks.SVOM = '/missions/svom'
   }
-  if (useFeature('SVOM')) {
-    if (selectedFormat == 'voevent') {
-      NoticeTypes['SVOM'] = [
-        'gcn.notices.svom.voevent.grm',
-        'gcn.notices.svom.voevent.eclairs',
-        'gcn.notices.svom.voevent.mxt',
-      ]
-    }
+  if (selectedFormat == 'voevent') {
+    NoticeTypes['SVOM'] = [
+      'gcn.notices.svom.voevent.grm',
+      'gcn.notices.svom.voevent.eclairs',
+      'gcn.notices.svom.voevent.mxt',
+    ]
```

Comment:
I am not certain that it is safe to modify a global variable in a component. Can you avoid that by making a new object from it?

Reply:
@lpsinger something like this latest change?

Reply:
👍